WiinUPro/Windows/MainWindow.xaml.cs

```diff
@@ -370,9 +370,66 @@ private void settingSuppressLostConn_Checked(object sender, RoutedEventArgs e)
         {
             AppPrefs.Instance.suppressConnectionLost = settingSuppressLostConn.IsChecked ?? false;
         }
+        private void settingAutoAddXInputDevices_Checked(object sender, RoutedEventArgs e)
+        {
+            AppPrefs.Instance.autoAddXInputDevices = settingAutoAddXInputDevices.IsChecked ?? false;
+        }
 
         #endregion
 
+        public void AddXInput(bool[] rumbleSub)
+        {
+            int n = 3;
+            for (; n > -1; n--)
+            {
+                if (rumbleSub[n])
+                {
+                    Console.WriteLine(n.ToString());

```

Comment:
I apparently forgot to remove this Console.WriteLine.

Reply:
Whoops.
This method can be simplified a bit but so could a lot of the code in the project, so this is good for now.